actors/builtin/methods.go

```diff
@@ -25,14 +25,19 @@ const (
 	Method_RewardActor_AwardBlockReward = MethodPlaceholder + iota
 )
 
-const (
-	Method_MultiSigActor_Propose = MethodPlaceholder + iota
-	Method_MultiSigActor_Approve
-	Method_MultiSigActor_AddAuthorizedParty
-	Method_MultiSigActor_RemoveAuthorizedParty
-	Method_MultiSigActor_SwapAuthorizedParty
-	Method_MultiSigActor_ChangeNumApprovalsThreshold
-)
+type msMethods struct {
+	Constructor                 abi.MethodNum
+	Propose                     abi.MethodNum
+	Approve                     abi.MethodNum
+	Cancel                      abi.MethodNum
+	ClearCompleted              abi.MethodNum
```

Comment:
Multisig does automatic cleanup, given that I think we can remove this method.

Reply:
I would keep it here, until it is removed from https://github.com/filecoin-project/specs/blob/e331e26c185b8685eb52bb1b9df4e5519b43f38a/src/systems/filecoin_token/multisig/multisig_actor.id

Reply:
Ugh, that file shouldn't exist.